### PR TITLE
upgrading: completed

### PR DESCRIPTION
**Pull request by JoshSummerTop**
@cbeck

### CODE REVIEW

File: app/controllers/charges_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    redirect_to user_path(current_user)
+
+    rescue Stripe::CardError => e
+      flash[:alert] = e.message
+      redirect_to new_charge_path
+  end
+
+  def destroy
+    @user = current_user
```
Comment (cbeck) [inline app/controllers/charges_controller.rb]: Not necessary.

File: app/controllers/charges_controller.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    if @user.standard!
```
Comment (cbeck) [inline app/controllers/charges_controller.rb]: Ok, that downgrades the user, but not the wikis. In this method, you need this:

```
current_user.wikis.update_all(private: false)
```

File: app/controllers/charges_controller.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      flash[:notice] = "You're membership has changed to Standard."
+      redirect_to @user
```
Comment (cbeck) [inline app/controllers/charges_controller.rb]: So it would be redirect to current_user.

**Comment by cbeck** [comment]
I keep seeing where you reassign current_user to @user. This is not necessary with devise. current_user is available in all controllers, helpers and views. Just use that.

**Comment by JoshSummerTop** [comment]
@cbeck. Should be better now.

**Pull request merged**